examples/simple-agent/src/utils.js

```diff
@@ -1,10 +1,22 @@
 import getRawBody from 'raw-body';
+import { DataStoreLevel, Dwn, MessageStoreLevel } from '@tbd54566975/dwn-sdk-js';
 import { Web5 } from '@tbd54566975/web5';
 import fs from 'node:fs';
 import mkdirp from 'mkdirp';
 import { createRequire } from 'node:module';
 
-const web5 = new Web5();
+// Use custom names for the block, message, and data stores to make it possible to launch multiple simple agents
+// in the same directory.  If you don't do this, you will get LevelDB lock errors.
+const port = await getPort(process.argv);
+const dataStore = new DataStoreLevel({ blockstoreLocation: `DATASTORE-${port}` });
+const messageStore = new MessageStoreLevel({
+  blockstoreLocation : `MESSAGESTORE-${port}`,
+  indexLocation      : `INDEX-${port}`,
+});
```

Comment:
i believe we'll need to do the same for `eventlog` and `index`

Reply:
`INDEX` path was already being set... when defining `messageStore`:

```javascript
const messageStore = new MessageStoreLevel({
  ...
  indexLocation      : `INDEX-${port}`,
});
```

Setting EventLog path addressed in 6d63a0ba443e120f9ffcc0be5020601dbea4f67f